tests/functional/spec/textfield_suggestions.spec.js

```diff
@@ -8,3 +9,22 @@ describe("Suggestions", () => {
     expect($$(".js-autosuggest-listbox li").length).to.not.equal(0);
   });
 });
+
+describe("Suggestions", () => {
+  it("Given I open a textfield with a suggestions url that allows multiple suggestions, when I have entered text and picked suggestion from a list, then after typing more text it will show new suggestions", () => {
+    browser.openQuestionnaire("test_textfield_suggestions.json");
+    $(SuggestionsPage.country()).setValue("United States of America");
+    $(SuggestionsPage.submit()).click();
+    $(MultipleSuggestionsPage.multipleCountry()).click();
+    browser.keys("Ita");
+    $("#multiple-country-answer-listbox li").waitForDisplayed();
+    expect($$(".js-autosuggest-listbox li").length).to.equal(2);
+    $("#multiple-country-answer-listbox__option--0").click();
+    $(MultipleSuggestionsPage.multipleCountry()).click();
+    // Browser needs to pause before typing starts, otherwise this sequence is executed almost instantly
+    browser.pause(500);
+    browser.keys(" United");
+    $("#multiple-country-answer-listbox li").waitForDisplayed();
+    expect($$(".js-autosuggest-listbox li").length).to.equal(3);
```

Comment:
Fixing on the number of suggestions displayed makes the test fragile, and it shouldn't be needed as all we're concerned with is whether the suggestions are displayed or not.

Reply:
It checks if list of suggestions is visible now and if one of suggestions can be picked. For some reason functional tests cannot get list of passport countries during build process and fails.